pages/index.tsx

```diff
@@ -38,3 +40,21 @@ export default function HomePage() {
     </Layout>
   );
 }
+
+export const getServerSideProps: GetServerSideProps = async (_context) => {
+  let packageCount;
+  try {
+    const response = await fetch(`${API_URL}/api/packages?limit=${Number.MAX_SAFE_INTEGER}`, {
+      method: 'HEAD'
+    });
+    packageCount = parseInt(response.headers.get('x-total-count'));
+    packageCount = null;
+  } catch (error) {
+    packageCount = null;
+  }
+  return {
+    props: {
+      packageCount: packageCount || 0
```

Comment:
Here you don't need a default value, you can just return `packageCount` directly